src/appwindow.cpp

```diff
@@ -386,7 +387,7 @@ void AppWindow::openTab(MainWindow *window)
     onEditorFileChanged();
 }
 
-void AppWindow::openTab(const QString &path)
+void AppWindow::openTab(const QString &path, const QString &lang)
```

Comment:
To provide `new py` command that opens a new tab with the given language. openTab should be able to open a Tab with the language specified. If it is empty, the lang is detected from path suffix.